libs/intro/generate/helpers/slides.ts

```diff
@@ -0,0 +1,30 @@
+import { defer, forkJoin, from, ObservableInput } from 'rxjs';
+import { map, switchMap, switchMapTo } from 'rxjs/operators';
+
+import { CODELAB_PRESENTATION_ID } from './const';
+import { gSlides, validateToken$ } from './utils';
+
+export const getSlides$ = validateToken$.pipe(
```

Comment:
This is a bit clunky - use validateToken$, and then gSlides, why not export just single observable - gSlides$?